Tests/PostgreSQLTests/PostgreSQLConnectionTests.swift

```diff
@@ -342,6 +342,25 @@ class PostgreSQLConnectionTests: XCTestCase {
         _ = try categories.wait()
     }
 
+    func testNotifyAndListen() throws {
+        let notifyConn = try PostgreSQLConnection.makeTest()
+        let listenConn = try PostgreSQLConnection.makeTest()
+        let channelName = "Foo"
+        let messageText = "Bar"
+        var messageReceived = false
+        
+        try listenConn.listen(channelName) { text in
+            messageReceived = text == messageText
+        }.catch({ err in XCTFail("error \(err)") })
+
+        try notifyConn.notify(channelName, message: messageText).wait()
```

Comment:
I would suggest sending two or three notifications, just to ensure that receiving more than one notification works.